### DISCUSSION TITLE
Best practice on "publisherDefaults"

### DISCUSSION THREAD

**Discussion by mrifni** [readium/kotlin-toolkit#47] [Q&A]
Whats like the best practice on whether to enable "publisherDefaults" to true or leave it off and let the user adjust it.
So far Ive found making "publisherDefaults" enabled works better for the ebooks I have , with it disabled the texts are shrunk and difficult to read.
need some suggestions from the experts on this.

**Comment by mickael-menu** [comment]
You could have nicer results by enabling publisher's defaults by default.

However, letting the user change this setting is important because changing the font size only work with publisher's defaults off, for some poorly-authored books using absolute font sizes. Some reading apps choose to disable publisher's defaults right away to avoid confusing users with the font size.

Also, if you want to change [any of the Readium CSS settings requiring the flag `--advancedSettings`](https://github.com/readium/readium-css/blob/master/docs/CSS19-api.md), publisher's defaults need to be disabled (= advanced settings enabled).

As you can see, it's not a clear-cut choice, but to summarize:
* If you let the user change this, you can set `publisherDefaults` to true by default.
* If the user can't change it, set it to `false`, otherwise it breaks changing the font size for some books.
